src/components/FeatureViews/Registration/RegistrationCart/RegistrationCartContainer.js

```diff
@@ -67,284 +67,285 @@ const GET_COURSES_AND_STUDENTS_TO_REGISTER = gql`
+                registrationCart,
+                currentParent,
+                updateSession,
+            }}>
+            <BackgroundPaper>
+                <Grid container>
+                    <RegistrationActions />
                 </Grid>
-            </>
-             : 
-              <PaymentBoard />
-            }
-          </Grid>
-        </Grid>
-        <Dialog
-          open={parentRegistrationConfirmation}
-          onClose={() => setParentConfirmation(false)}
-        >
-          <DialogTitle>
-            <Grid
-              container
-              direction="row"
-              justify="center"
-              alignItems="center"
-            >
-              <Grid item>
-                <CheckCircleIcon color="primary" style={{ fontSize: "3em" }} />
-              </Grid>
-              <Grid item>
+                <hr />
+                <Typography align="left" variant="h2">Registration Cart</Typography>
                 <Typography
-                  color="primary"
-                  align="center"
-                  style={{
-                    fontSize: "1.2em",
-                    fontWeight: 500,
-                    padding: "15px",
-                  }}
-                >
-                  Request Saved
+                    align="left"
+                    data-cy="payment-title"
+                    gutterBottom
+                    style={{"fontSize": "2em"}}>
+                    Pay for Course(s)
                 </Typography>
-              </Grid>
-            </Grid>
-          </DialogTitle>
-          <DialogContent>
-            <Typography align="center">
-              Your request has been submitted!
-            </Typography>
-            <Typography align="center">
-              Here’s the next step to complete the enrollment process:
-            </Typography>
+                <Grid container item>
+                    <Grid container direction="row" spacing={5}>
+                        {
+                            Object.entries(registrationCart)
+                                .map(([studentId, registration]) =>
+                                    (<StudentRegistrationEntry
+                                        key={studentId}
+                                        registrationList={registration}
+                                        student={studentData.find((student) => student.user.id === studentId)} />))
+                        }
+                    </Grid>
+                    <Grid alignItems={parentIsLoggedIn && "flex-end"} container
+                        direction={parentIsLoggedIn ? "column" : "row"}
+                        item
+                        justify={parentIsLoggedIn ? "flex-end" : "space-between"}
+                        spacing={parentIsLoggedIn && 4}
+                        style={{"marginTop": "50px"}}>
+                        {parentIsLoggedIn ?
+                            <>
+                                <Grid item xs={5}>
```

Comment:
If you remove the parent access control, you can refactor the code from line 252 - 276 to handle the pay later flow instead